### ISSUE TITLE
v 1.1.0 is not backward compatible

### ISSUE THREAD

**Issue by nkhanna09** [ActiveJpa/activejpa#46]
Hey,

You changed the visibility of filter, conditions classes and coz of which its not compatible with older version.

**Comment by ganeshs** [comment]
@nkhanna09 yes. This should have been a major version upgrade. Apologies for the miss. The release note below captures the changes. 

https://github.com/ActiveJpa/activejpa/releases/tag/1.1.0
